src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -466,18 +505,15 @@ public IndexModel describeIndex(String indexName) throws PineconeException {
      * @param indexName The name of the index to configure.
      * @param podType The new podType for the index. Can be null if not changing the pod type.
      * @param replicas The desired number of replicas for the index, lowest value is 0. Can be null if not changing the number of replicas.
+     * @param deletionProtection Enable or disable deletion protection for the index.
      * @return {@link IndexModel} representing the configured index.
      * @throws PineconeException if an error occurs during the operation, the index does not exist, or if any of the arguments are invalid.
      */
-    public IndexModel configureIndex(String indexName, String podType, Integer replicas) throws PineconeException {
+    public IndexModel configurePodsIndex(String indexName, String podType, Integer replicas, DeletionProtection deletionProtection) throws PineconeException {
```

Comment:
The deletion protection feature works on both serverless and pod indexes. So it seems like an additional configure method is needed for serverless.

Reply:
Added here: https://github.com/pinecone-io/pinecone-java-client/pull/141/files#diff-266ed57e8c66ca7cb9b5bb7c97c25d293555998fbc6c8d50f6f70c80da53c4e3R623